config/pipeline.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+    AttributeDefinitions: [ /* required */
+        {
+            AttributeName: 'id', /* required */
+            AttributeType: 'S' /* required */
+        },
+        {
+            AttributeName: 'scmUrl', /* required */
+            AttributeType: 'S' /* required */
+        }
+    ],
+    KeySchema: [ /* required */
+        {
+            AttributeName: 'id', /* required */
+            KeyType: 'HASH' /* required */
+        }
+    ],
+    ProvisionedThroughput: { /* required */
+        ReadCapacityUnits: 1, /* required */
+        WriteCapacityUnits: 1 /* required */
+    },
+    TableName: 'Pipelines', /* required */
+    GlobalSecondaryIndexes: [
+        {
+            IndexName: 'ScmUrls', /* required */
```

Comment:
Convenience of debugging.

The index is on SCM Urls. Take the scenario where we label the index name as "scmUrl". If the error states that "scmUrl" is invalid, then what's the issue: the index or the column?